svcomp/bench/toInstallDir/runSMACKBench.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Not sure where to leave this comment. I don't particularly like the name of this folder (toInstallDir). Why did you call it like that?
If nothing else, then I would at least probably prefer to-install-dir.